src/appwindow.cpp

```diff
@@ -345,11 +345,11 @@ void AppWindow::maybeSetHotkeys()
     }
 }
 
-bool AppWindow::closeTab(int index)
+bool AppWindow::closeTab(int index, bool noConfirmQuit = false)
```

Comment:
In order to handle `q!` (forcefully close). We should have a way to close the window without the confirmation. This argument has been added to handle that.